client/components/Contact.js

```diff
@@ -80,48 +96,36 @@ class Contact extends Component {
       firstName,
       lastName,
       email,
-      comment
+      comment,
+      interest
     } = this.state;
 
-    /* options: [
+    const options = [
       {
-        value: 'Volunteer',
-        label: this.props.intl.formatMessage({
-          key: 'volunteer',
-          id: 'components.Contact.fields.interest.options.volunteer',
-          defaultMessage: 'Volunteer',
-          description: "'Volunteer' option for the Interest field in the Contact form"
-        })
+        value: 'volunteer',
+        label: 'Volunteer to help',
```

Comment:
Can you change the labels to use `this.props.intl.formatMessage`? You should just be able to copy and paste them from here.

Reply:
I'm not familiar with i18n. Do you think you can take care of the labeling after merging?

Reply:
Can you just copy the `label: this.props.intl.formatMessage({...` over the `label: '...`? Seems like that would be easier than going back and restoring the formatMessage call.

Reply:
No, because not all of the messages exist in the i18n translations. That's why I would rather have you do it altogether. The rest of the fields and text will also need translations so I think it would be easier to just do translations and anything associated with i18n in a different PR. This is also because I am not familiar with i18n so I would rather not touch it right now.